LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2007-2017 Citation Style Language
```

Comment:
See suggestion by @adam3smith in https://github.com/citation-style-language/schema/pull/143#issuecomment-428329245

> Copyright (c) 2007-2018 Citation Style Language and Contributors